Source/Peripheral+Convenience.swift

```diff
@@ -136,6 +143,7 @@ extension Peripheral {
      */
     public func monitorValueUpdate(for identifier: CharacteristicIdentifier) -> Observable<Characteristic> {
         return characteristic(with: identifier)
+            .asObservable()
```

Comment:
Why do we need `asObservable()` here?

Reply:
This one is necessary because `monitorValueUpdate` is an infinite stream and therefore must be an `Observable`

Reply:
Right, makes sense. Thanks!